<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>peronaMalikDialog</class>
 <widget class="QDialog" name="peronaMalikDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Perona-Malik</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>NSteps:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>diffusion parameter (K):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="timeStepSpinBox">
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="stepType">
      <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
     </property>
     <property name="value">
      <double>0.200000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="nStepsSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>solver scheme:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>time step:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="kParamSpinBox">
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="schemeComboBox">
     <item>
      <property name="text">
       <string>explicit</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>regularized (semi-implicit)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="overrideCheckBox">
     <property name="text">
      <string>override time step</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>peronaMalikDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>peronaMalikDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>schemeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>peronaMalikDialog</receiver>
   <slot>bindTimeStep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>overrideCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>peronaMalikDialog</receiver>
   <slot>overrideTimeStep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>bindTimeStep()</slot>
  <slot>overrideTimeStep()</slot>
 </slots>
</ui>
